content/survey_2022.ipynb

```diff
@@ -0,0 +1,4709 @@
+{
```

Comment:
<p>Please disable the zoom scroll and pan tools by default on these plots if you can; hover is useful but the rest are just distracting and too easy to invoke accidentally.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>